operator/pkg/ciliumendpointbatch/manager.go

```diff
@@ -54,30 +54,29 @@ type cebManager interface {
 	// manages CEB's.
 	getCebFromCache(cebName string) (*cilium_v2.CiliumEndpointBatch, error)
 	getCebCopyFromCache(cebName string) (*cilium_v2.CiliumEndpointBatch, error)
-	findCeb() *cebTracker
 	updateCebInCache(ceb *cilium_v2.CiliumEndpointBatch, deepCopy bool)
 	deleteCebFromCache(cebName string)
 	getRemovedCeps(string) map[string]struct{}
 	clearRemovedCeps(string, map[string]struct{})
 	createCeb(cebName string) *cebTracker
+	addCEPtoCEB(ceb *cilium_v2.CoreCiliumEndpoint, cebName string)
 	// APIs to collect metrics of CEB and CEP
 	getTotalCepCount() int
 	getCepCountInCeb(cebName string) int
 	getCebCount() int
 	getAllCepNames() []string
 }
 
-// Implementation of FirstComeFirstServe batching mode. If new CEP is inserted,
-// then the CEP is queued in any one of the available CEB. CEPs are inserted into
-// CEBs without any preference or any priority.
-type cebManagerFcfs struct {
+// cebMgr is used to batch CEP into a CEB, based on FirstComeFirstServe. If a new CEP
```

Comment:
Comment is a bit misleading, ceMgr should be the common piece, and cebManagerFcfs later is for FCFS only.

Reply:
Comment isn't misleading. It stating the fact, `cebMgr` implements batching based on FCFS.  In case of `cebManagerFcfs` default behavior of `cebmgr` suits well.

Reply:
Yea, I agree the comment agrees with the code, my point is this code structure is confusing in the first place, the identityManager has an anonymous field `cebMgr` which in current structure implements FCFS, so when I read the code, it reads to me as if IdentityManager is a "superset" of FCFS manager.

A better organization should be to keep the common pieces in `cebMgr` then the different behaviors should be implemented in corresponding structs respectively.

Reply:
`IdentityManager` have an embedded struct `cebMgr` , which borrows whole functionality of `cebMgr` implementation.  It replaces couple of functions from base `cebMgr`, that are specific to `Identity` based batching.

Reply:
I understand that, I'm just confused if `cebMgr` is in fact the FCFS mgr, why having [1]? What's the special handling in the comment refering to?

Reply:
Technically, we don't need `cebManagerFcfs` struct. I kept that as placeholder, in future if we need special handling for `cebManagerFcfs`, suppose start filling from CEP's in CEB, from lower CEP counts.

Reply:
I see, maybe add a comment in [1] stating FCFS is expected to be the base manager by design (which is still confusing IMHO, because our default is identify batching).

Reply:
Done.